.github/workflows/release-homebrew.yaml

```diff
@@ -14,5 +14,5 @@ jobs:
         tap: microsoft/git
         name: git-credential-manager-core
         type: cask
-        releaseAsset: gcm-osx-x64-(.*)\.pkg
+        releaseAsset: gcm-osx-(?<runtime>.*)-(?<version>.*)\.pkg
         alwaysUsePullRequest: true
```

Comment:
Remember we'll need to update the action version once https://github.com/mjcheetham/update-homebrew/pull/10 is done and a release made :wink: